Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/AnnexB/Index.cshtml

```diff
@@ -1,7 +1,7 @@
 @page "/task-list/{id:int}/annex-b-form-saved"
 @model Dfe.PrepareConversions.Pages.AnnexB.IndexModel
 @{
-   ViewData["Title"] = "Confirm Annex B received";
+   ViewData["Title"] = "Annex B form";
```

Comment:
Why we are changing the title again ?? If this is a new requirement then we need to update the cypress test `annex-b-form-tab.cy.js` (line 18) as well